pulp_python/tests/functional/api/test_sync.py

```diff
@@ -336,3 +338,21 @@ def test_sync_provenance(python_repo_with_sync, python_remote_factory, python_co
     summary = python_content_summary(repository_version=repo.latest_version_href)
     assert summary.present["python.python"]["count"] == 2
     assert summary.present["python.provenance"]["count"] == 2
+
+
+@pytest.mark.parallel
+def test_package_sync_with_metadata(
+    pulp_content_url,
+    python_distribution_factory,
+    python_remote_factory,
+    python_repo_with_sync,
+):
+    """
+    Test that the sync of a Python wheel package creates a metadata artifact.
+    """
+    remote = python_remote_factory(includes=PYTHON_XS_PROJECT_SPECIFIER)
```

Comment:
We should choose a different package since this test runs in parallel meaning the upload tests will also be creating the same content. All the wheels on the fixtures should have the metadata file available.